.github/workflows/run-onert-android-build.yml

```diff
@@ -69,8 +68,8 @@ jobs:
           pip3 install numpy
           sudo apt-get update && sudo apt-get -qqy install scons
 
-      # Use NDK 26.3.11579264
+      # Use NDK Default
       - name: Build onert
         run: |
-          export NDK_DIR=/usr/local/lib/android/sdk/ndk/26.3.11579264
+          export NDK_DIR=${ANDROID_NDK}
```

Comment:
Just for note:

https://github.com/actions/runner-images/blob/main/images/ubuntu/Ubuntu2204-Readme.md#environment-variables-2